pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -148,19 +148,21 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 		Expect(resp.Allowed).To(BeTrue())
 	})
 
-	DescribeTable("path validation should fail", func(path string) {
-		Expect(validatePath(k8sfield.NewPath("fake"), path)).To(HaveLen(1))
```

Comment:
its ok that we dont want to use an unexported function to test behavior instead of implementation, but I believe you should still make sure the admition fails on what you expect (I saw my fair amount of tests checking for admit true or false and in the end the false returned because of something else and not what was expected)
Maybe you can a check to validate the expected respond message

Reply:
Fair point, I'll change it.